sql/hive-thriftserver/v2.3.5/src/main/java/org/apache/hive/service/cli/operation/SQLOperation.java

```diff
@@ -373,7 +373,7 @@ private RowSet decodeFromString(List<Object> rows, RowSet rowSet)
     int protocol = getProtocolVersion().getValue();
     for (Object rowString : rows) {
       try {
-        rowObj = serde.deserialize(new BytesWritable(((String)rowString).getBytes("UTF-8")));
+        rowObj = serde.deserialize(new BytesWritable(((String)rowString).getBytes(StandardCharsets.UTF_8)));
       } catch (UnsupportedEncodingException e) {
```

Comment:
Do we still need this `UnsupportedEncodingException`?

Reply:
Ur, @HeartSaVioR . Could you test this again, please?
For me, it's used without a proper `import`. This will fail at compilation.

Reply:
Yes, here and above I think this can no longer throw UnsupportedEncodingException